sdk/src/types/block/semantic/mod.rs

```diff
@@ -31,12 +32,16 @@ pub struct SemanticValidationContext<'a> {
     pub(crate) unlocks: Option<&'a [Unlock]>,
     pub(crate) input_amount: u64,
     pub(crate) input_mana: u64,
+    pub(crate) mana_rewards: BTreeMap<OutputId, u64>,
+    pub(crate) reward_context_inputs: HashMap<OutputId, RewardContextInput>,
+    pub(crate) commitment_context_input: Option<CommitmentContextInput>,
+    pub(crate) bic_context_input: Option<BlockIssuanceCreditContextInput>,
```

Comment:
There can be more than one, but I can change that in my PR

Reply:
alright!